src/controllers/api/clients/bookings.js

```diff
@@ -0,0 +1,67 @@
+import models from '../../../database/models';
+import { getToken } from '../../create_token'
+
+export const getBookings = (req, res) => {
+  models.sequelize.query("SELECT ext_bookings.event_title, bookings.start_at, bookings.end_at,bookings.room_id, events_courses.title FROM ext_bookings RIGHT OUTER JOIN bookings ON bookings.id = ext_bookings.booking_id LEFT OUTER JOIN events_courses ON events_courses.id = bookings.event_id").spread((results, metadata) => {
+    const resObj =  results.map((booking) => {
+      return Object.assign(
+        {},
+        {
+          id: booking.id,
+          start: booking.start_at,
+          end: booking.end_at,
+          resourceId: booking.room_id,
+          title: booking.title || booking.event_title
+        }
+      );
+   });
+   res.json(resObj);
+  })
+};
+
+export const postBookings = (req, res) => {
+  const { token } = req.cookies;
```

Comment:
instead of doing this, when the user login, you can set something like 
`req.user ={start_at, end_at,room_id, name, phone,email,event_title}`
so this let you to do it once

Reply:
I don't understand how this would help to do it at once?
the booking process goes in three phases 
- request to '/phone/verification/ start', this will send a code to the user phone number and the data that the user inserted it'll be stored in the cookies "the request will end here"
- then after the user gets the code in SMS he will enter it and a POST request will be sent to : 
  - '/api/v1/booking' if it was a booking operation and here we use the cookies to retrieve the data that the user sent in the previous request
  - or to 'api/v1/courses' if it was a course registration and same will happen with the data